docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: "3"
```

```suggestion
version: "2.1"
```
faz essa alteração aqui